flink-ml-lib/pom.xml

```diff
@@ -106,6 +112,11 @@ under the License.
       <type>jar</type>
       <scope>test</scope>
     </dependency>
+      <dependency>
+          <groupId>com.google.code.gson</groupId>
+          <artifactId>gson</artifactId>
+          <version>2.8.6</version>
+      </dependency>
```

Comment:
`ReadWriteUtils` has provided methods to generate json. Shall we reuse the existing method and avoid adding new dependencies?

Reply:
ReadWriteUtils may not satisfy my need, it only a utils to help serialize model data.

Reply:
For the json tool ReadWriteUtils supply not support some class's serialization. Here, I use the google replace the default json tool. Then, we have only one json tool.